### PR TITLE
Revert changes for admin server IP detection

### PR DESCRIPTION
**Pull request by rhafer**
The changes to derive the admin server IP from the DHCP server information in the Win32_NetworkAdapterConfiguration object don't seem to work reliably when the windows nodes have more than one Network Card and the admin network is not connected to the first of those cards. This simply reverts those commits and goes back to setting in statically during build time.

Should fix: https://bugzilla.novell.com/show_bug.cgi?id=866797

### CODE REVIEW

**Comment by rhafer** [comment]
Obviously the better way to fix this would be to fix the DHCPServer detection. I just don't know how to do that. In a test setup with a 6 NIC hyper-v node we got back this list of DHCP Servers: `255.255.255.0 255.255.255.0 10.125.1.1 255.255.255.0 255.255.255.0 255.255.255.0`
So I guess it would just be a matter of filtering out the `255.255.255.0` ones. @ociuhandu any idea?

**Comment by ociuhandu** [comment]
We're currently looking into this and we'' get back with the fix asap. @rhafer

File: build_winpe.ps1 [hunk 1]
```diff
@@ -186,11 +186,11 @@ popd
 Add-Content $startnet_cmd "`n"
 if($built_for_crowbar)
 {
-  Add-Content $startnet_cmd "`n powershell `"`$DHCPServer = @(Get-WMIObject -Class Win32_NetworkAdapterConfiguration -Filter IPEnabled=True -ComputerName . | Select-Object -Property DHCPServer).DHCPServer[0] ; Write-Host DHCPServer=`$DHCPServer ; net use $crowbar_mountpoint \\`$DHCPServer\$crowbar_share`""
+  Add-Content $startnet_cmd "`n net use $crowbar_mountpoint \\$crowbar_server_ip\$crowbar_share"
```
Comment (ociuhandu) [inline build_winpe.ps1:189]: The fix should be:
$DHCPServers=@(gwmi Win32_NetworkAdapterConfiguration -Filter IPEnabled=True | where {$_.DHCPServer -and !($_.DHCPServer -match "^255")}); if ($DHCPServers) {$DHCPServer=$DHCPServers[0].DHCPServer} else {throw "DHCP Server Not Found"}

Comment (vuntz) [inline build_winpe.ps1:189]: Thanks, I submitted https://github.com/cloudbase/adk-tools/pull/8 so it's easier for someone to try this code.

**Comment by ociuhandu** [comment]
dropped due to https://github.com/cloudbase/adk-tools/pull/8

**Pull request closed without merging**